docs/pages/access-controls/guides/impersonation.mdx

```diff
@@ -127,7 +127,7 @@ $ tctl auth sign --user=jenkins --format=openssh --out=jenkins --ttl=240h
 ```
 
 </TabItem>
-<TabItem scope={["cloud"]} label="Teleport Enterprise Cloud">
+<TabItem scope={["cloud"]} label="Teleport Enterprise">
```

Comment:
Would it make sense to remove these changes from this PR to reduce the size of the diff? If we do want to change these tabs components, I'm not sure we even need them to be tabbed, since the only difference is an example URL.